README.md

```diff
@@ -1 +1,125 @@
-# Hello-MV-Project
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+ - [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 [Hello-Microverse] <a name="about-project"></a>
+
+> This is my first MV Project as a Micronaut. In this project, I created a branch and applied linters.
+
+## 🛠 Built With <a name="built-with"></a>
+* HTML 
+* CSS
+### Tech Stack <a name="tech-stack"></a>
+> Hello Microverse!
+> ### Key Features <a name="key-features"></a>
+
+- **Using linter**
+- **Using gitHub flow**
+- **Creating a branch**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Here is my first project as a Micronaut to understand setting up linter.
+
+### Prerequisites
+
+In order to run this project you need: to download it to your desired PC.
+
+### Setup
+
+Clone this repository to your desired folder:
+1. Click Code button (shown in green color)
+2. Copy URL.
+3. Please clone it with the following command: git clone git@github.com:myaccount/my-project.git
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
```

Comment:
- [ ] All lines beginning with `>` should be replaced.  The symbol `>` marks a line as a comment and they are only used to provide guidance to you when filling in the `README`. Kindly remove the `>` from your readme 😄

Reply:
Please let me know how I can change title. (Pull request for set-up-mv branch)

Reply:
Hello @mryeminthu, Changing the PR title is not difficult at all. Go to Pull request on GitHub and look at the title, to the extreme right of this title you should see the edit button. click it and edit your repo. A save button will also appear at about the same position & you should save once you have made the change. Please reach out to me on zoom or slack if you need further clarification 👍🏾